CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ the release.
   ([#1515](https://github.com/open-telemetry/opentelemetry-demo/pull/1486))
 * [frontend] Fix imageloading issues on optimized images. bump next.js version
   ([#1571](https://github.com/open-telemetry/opentelemetry-demo/pull/1571))
+* [cartservice] bump .NET package to 1.8.1 release
```

Comment:
cosnider putting it near entry for 1.8.0 release, or just replace both entries by

```md
* [cartservice] update .NET package to 1.8.1 release
  ([#1514](https://github.com/open-telemetry/opentelemetry-demo/pull/1514), [#1571](https://github.com/open-telemetry/opentelemetry-demo/pull/1571))
```

Reply:
Good point, entries merged in https://github.com/open-telemetry/opentelemetry-demo/pull/1580/commits/0a01213f9a1a7279646367046acd851fe30d1ddc